boot.js

```diff
@@ -19,18 +19,9 @@
   scriptNode.src = buildSrc;
   scriptNode.type = 'text/javascript';
   document.body.appendChild(scriptNode);
-  // create theme override node in body
-  const themeOverridesSrc = (basename && basename !== '/') ? `${basename}/src/css/themeoverrides.css?_=${cssVersion}` : `/src/css/themeoverrides.css?_=${cssVersion}`;
-  let linkNode = document.createElement('link');
-  linkNode.href = themeOverridesSrc;
-  linkNode.type = 'text/css';
-  linkNode.id = 'gen3-theme-overrides';
-  linkNode.rel = 'stylesheet';
-  document.body.appendChild(linkNode);
```

Comment:
removed this because i don't think the relative file path is necessary

Reply:
i think part of this logic is still needed because it will forces the browser to reload the css files each time when there is a change

Reply:
put it back

Reply:
on further inspection this loads the css file from the server, you need to rerun the dev script to see your changes as it has to copy the file so its never going to have up to date changes. Should i leave this code in or take it out? Im thinking removing it is  the better call so locale development does not get confused if they are seeing server code or local code

Reply:
yeah so if we remove this section them basically in local dev mode we will only be loading `themeoverrides.css` from local side, and not from remote side right?
I think that is a good idea and we should be good toi remove them then

Reply:
thats correct, ill run some tests in different environments to make sure its good but thats how it should work